### ISSUE TITLE
Support `eslint-plugin-vue`'s disable comments

### ISSUE THREAD

**Issue by FloEdelmann** [eslint-community/eslint-plugin-eslint-comments#212]
In Vue Single File Components (`*.vue` files), "HTML comments" can be used to disable ESLint rules: https://eslint.vuejs.org/rules/comment-directive.html

```vue
<template>
  <!-- eslint-disable-next-line vue/max-attributes-per-line -->
  <div a="1" b="2" c="3" d="4" />
</template>
```

Unfortunately, those comments are not detected by `eslint-plugin-eslint-comments`.

*(Note: This issue is the same as #136, which was auto-closed as stale without a warning, but is still relevant.)*

**Comment by GrantGryczan** [comment]
This applies to CSS linting in Vue too, by the way.

```vue
<template></template>

<style module>
// eslint-disable-next-line vue-scoped-css/require-selector-used-inside
.unused {
  filter: brightness(0);
}
</style>
```

This should warn of the missing comment.

**Comment by KonstantinFilipov** [comment]
I had the same need - reporting the use of HTML comments inside .vue files.
Unfortunately, it seems the creator may have had different intentions, so someone will need to evaluate exactly what changes should be made.
Here is my fix: https://github.com/eslint-community/eslint-plugin-eslint-comments/blob/dbe7edbbae394c21883f49e1def5ecc1831fcdca/lib/internal/get-all-directive-comments.js#L61-L112

```js
    return result.concat(
        sourceCode
            .getInlineConfigNodes()
            .concat(sourceCode.ast.templateBody.comments) // <-- add line: to append all html comments to the result
            .map((node) => ({
                node,
                range: sourceCode.getRange(node),
            }))
            .filter(
                ({ range }) =>
                    // The node has intersection of the directive comment.
                    // So, we need to skip it.
                    !result.some((comment) => comment.range[0] <= range[1] && range[0] <= comment.range[1]),
            )
            .map(({ node, range }) => {
                const nodeText = sourceCode.text.slice(range[0], range[1]);
                const commentValue = extractCommentContent(nodeText);
                const directiveComment = utils.parseDirectiveText(commentValue);

                return {
                    directiveComment,
                    node,
                    range,
                };
            })
            .filter(({ directiveComment }) => {
                return (
                    directiveComment != null &&
                        ['eslint-disable', 'eslint-disable-line', 'eslint-disable-next-line', 'eslint-enable']  // <-- change: we need the comments that include disable, not exclude, so "!" is removed
                        .includes(
                            directiveComment.kind,
                        )
                );
            })
            .map(
                ({ directiveComment, node, range }) =>
                /** @type {DirectiveComment} */ ({
                    kind: directiveComment.kind,
                    value: directiveComment.value,
                    description: directiveComment.description,
                    node,
                    range,
                    get loc() {
                    return sourceCode.getLoc(node);
                    },
                }),
            ),
    );
```

**Comment by Grawl** [comment]
This should work outside of `<template>` too, for example:

```vue
<template>
	<div></div>
</template>
<!-- eslint-disable-next-line vue/enforce-style-attribute -->
<style>
</style>
```

For now, this comment disables the rule for next line, but comments plugin is not warning me to describe this with `-- some explanation`.
